src/app/lib/config/services/persian.js

```diff
@@ -43,6 +43,36 @@ const persian = {
         callToActionLinkUrl: 'https://www.bbc.com/persian',
       },
     },
+    consentBanner: {
```

Comment:
Let's raise an issue to get Persian translations.

Reply:
Done :) https://github.com/bbc/simorgh/issues/1438